src/http/Service.cpp

```diff
@@ -78,9 +79,36 @@ Entry* Service::getConfigEntry(bool create)
     return NULL;
 }
 
-bool Service::isDatabaseOpened() const
+QList<Entry*> Service::getConfigEntries(bool create)
 {
-    if (DatabaseWidget* dbWidget = m_dbTabWidget->currentDatabaseWidget())
+    QList<Entry*> entries;
+
+    for (int i = 0; i < m_dbTabWidget->count(); i++)
```

Comment:
This `for` block should be wrapped with braces.

Reply:
That is inconsistent with the style of the rest of the file. Should I really *partially* fix the style of the file(s)? A separate code style commit would probably be better.

The same applies for NULL vs. nullptr.

Reply:
Older code don't follow the coding style in some places but all new PR's should stick to the project [style guide](https://github.com/keepassxreboot/keepassxc/blob/develop/.github/CONTRIBUTING.md#coding-styleguide).

Reply:
>That is inconsistent with the style of the rest of the file.

Reply:
>That is inconsistent with the style of the rest of the file

Older code, mostly before the fork, don't necessarily follow the current coding style, but all new PR's should stick to the project coding [style guide](https://github.com/keepassxreboot/keepassxc/blob/develop/.github/CONTRIBUTING.md#coding-styleguide).

>Should I really partially fix the style of the file(s)?

It's not required to fix the style where you didn't modify/add code, but we would appreciate if you do. :)

>A separate code style commit would probably be better.

Concur. I'm actually planning to submit a PR with these fixes in older code soon.

Reply:
Run this command to fix all the file codestyle to project standard
`clang-format -i -style=file src/http/Service.cpp` in the project root.

Yes, fix the codestyle in a new commit please